Configurations/descrip.mms.tmpl

```diff
@@ -439,13 +440,14 @@ all : build_sw build_docs
 test : tests
 {- dependmagic('tests'); -} : build_programs_nodep, build_modules_nodep copy-utils
         @ ! {- output_off() if $disabled{tests}; "" -}
-        DEFINE SRCTOP {- sourcedir() -}
-        DEFINE BLDTOP {- builddir() -}
-        DEFINE FIPSKEY {- $config{FIPSKEY} -}
+        DEFINE SRCTOP "$(SRCDIR)"
```

Comment:
What about [this location here](https://github.com/openssl/openssl/blob/018c5a1a9d/Configurations/descrip.mms.tmpl#L455-L457)? Should it be changed accordingly?

```
list-tests :
        @ ! {- output_off() if $disabled{tests}; "" -}
        @ DEFINE SRCTOP {- sourcedir() -}
```

https://github.com/openssl/openssl/blob/018c5a1a9d/Configurations/descrip.mms.tmpl#L457

Reply:
It's come to this, cleanup up MMS files. :)